lumpy-llamas/newsfeed/newsfeed_request.py

```diff
@@ -0,0 +1,52 @@
+import requests
+from requests.exceptions import HTTPError
+from multiprocessing.dummy import Pool
+THREAD_POOL = Pool(8)
+HACKERNEWS_URL = 'https://hacker-news.firebaseio.com/v0'
+
+
+# Function to get the new stories from the API, the output of this is a list with numbers
+# each number represents a new story, there are maximum 500 newstories
+def get_new_stories() -> list:
+    # newstories.json? refers to the new stories in the API
+    response = requests.get(f'{HACKERNEWS_URL}/newstories.json?')
+    return response.json()
+
+
+# Function to get the best stories from the API, the output of this is a list with numbers
+# each number represents a top story, there are maximum 500 best stories
+def get_best_stories() -> list:
+    # beststories.json? refers to the best stories in the API
+    response = requests.get(f'{HACKERNEWS_URL}/beststories.json?')
+    return response.json()
+
+
+def get_story(story_id):
+    api_response = requests.get(f'{HACKERNEWS_URL}/item/{story_id}.json').json()
+    print(api_response)
+    # if api_response.get("url") is not None:
```

Comment:
This debug print should be removed for the final product, along with that extra comment.